core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/NamespaceSynchronizationConfig.java

```diff
@@ -250,6 +285,32 @@ boolean addSynchronizedDocument(
     }
   }
 
+  @Nullable
+  DeleteManyModel<CoreDocumentSynchronizationConfig> removeSynchronizedDocuments(
```

Comment:
[nit] Maybe could call this `prepareRemoveSynchronizedDocuments` so as to indicate this won't remove them at this call site